lib/index.js

```diff
@@ -30,6 +42,9 @@ function createApp (opts) {
 
   app.get('/__bundle/:file.bundle.js', bundleFileRoute)
 
+  // Inject livereload snippet
+  app.use(require('connect-livereload')())
```

Comment:
Is this still needed with https://github.com/abacon/run-js/commit/d1790cb4a25ac093eafd6b570944f829df886f14?